Maple2.Model/Enum/QuestType.cs

```diff
@@ -4,12 +4,336 @@ public enum QuestType {
     EpicQuest = 0,
     WorldQuest = 1,
     EventQuest = 2,
-    DailyMission = 3,
-    FieldMission = 4,
+    DailyMission = 3, // Navigator
+    FieldMission = 4, // Exploration
     EventMission = 5,
     GuildQuest = 6,
     MentoringMission = 7,
     FieldQuest = 8,
     AllianceQuest = 9,
     WeddingMission = 10,
 }
+
+public enum QuestConditionType {
```

Comment:
seems unnecessary to dupe this enum (`AchievementConditionType`) just create a common one (`ConditionType`). There are also other places where this can be useful (I believe skills also have this condition enum)